src/aequitas/plot/bubble_disparity_chart.py

```diff
@@ -117,13 +117,33 @@ def __draw_metrics_rules(metrics, scales, concat_chart):
     return metrics_rules
 
 
+def __get_x_axis_values(x_domain, zero=True):
+    TICK_STEP_OPTIONS = [1, 2, 5, 10, 20, 50, 100]
+
+    def list_axis_values(limit, step):
```

Comment:
I don't have a great suggestion, but I think this function name should be more on the "create" side, rather than "list".

It's nothing major, particularly because of its very limited scope. But if you think of something...

Reply:
I was trying to avoid `get` 😅  I ended up on `list` as its also a verb and it's what we're returning.